gadget/update.go

```diff
@@ -433,6 +489,13 @@ type DiskVolumeValidationOptions struct {
 	// AllowImplicitSystemData has the same meaning as the eponymously named
 	// filed in EnsureLayoutCompatibilityOptions.
 	AllowImplicitSystemData bool
+	// ExpectedEncryptedPartitions is the a map of the names (gadget structure
+	// names) of partitions that are encrypted on the volume and what method
+	// they were encrypted with. This should only ever be ubuntu-data or
+	// ubuntu-save for now, but the map will indicate the name of the structure
+	// as the key and the type of encryption (currently only "LUKS") as the
+	// value in the map.
```

Comment:
If you need to touch this branch again, I recommend reformulating the last period (assuming that my understanding is correct) to something like this:

"For now, the only keys we should find in the map are "ubuntu-data" or "ubuntu-save", and the only type of encryption currently supported is LUKS."

The format of the map was already explained in the first period.

Reply:
thanks yes that is much simpler and easier to understand, thanks for the suggestion :-)